packages/publint/tests/playground.test.js

```diff
@@ -200,36 +198,48 @@ testFixture('invalid-repository-value-object-deprecated', [
  * @param {TestOptions} [options]
  */
 function testFixture(name, expectCodes, options) {
-  test(name, { concurrent: true }, async ({ expect }) => {
-    const fixtureName = name.replace(/\(.*$/, '').trim()
-    const pkgDir = path.resolve(process.cwd(), 'tests/fixtures', fixtureName)
-    const { messages } = await publint({
-      pkgDir,
-      level: options?.level,
-      strict: options?.strict,
-    })
-
-    // unfortunately the messages are not always in order as checks are ran in parallel,
-    // here we sort it to make the tests more consistent
-    messages.sort((a, b) => a.code.localeCompare(b.code))
-
-    if (options?.debug) {
-      const pkg = JSON.parse(
-        await fs.readFile(path.join(pkgDir, 'package.json'), 'utf-8'),
-      )
-      console.log()
-      console.log('Logs:', name)
-      messages.forEach((m) => console.log(formatMessage(m, pkg)))
-      console.log()
-    }
-
-    // you can test an array of objects
-    if (typeof expectCodes[0] === 'object') {
-      const codes = messages.map((v) => ({ code: v.code, type: v.type }))
-      expect(codes).toEqual(expectCodes)
-    } else {
-      const codes = messages.map((v) => v.code)
-      expect(codes).toEqual(expectCodes)
+  /** @type {import('vitest').TestOptions} */
+  const testOpts = { concurrent: true, timeout: process.env.CI ? 10000 : 5000 }
```

Comment:
I increased the timeout because the Windows tests were timing out. https://github.com/publint/publint/actions/runs/12854546717/job/35839037345

I also encountered a strange test failure where it looked like the fixtures were interfering with each other. https://github.com/publint/publint/actions/runs/12854649919/job/35839274199